OurUmbraco.Site/Views/Master.cshtml

```diff
@@ -71,11 +71,24 @@
 
             if (string.IsNullOrWhiteSpace(title) == false)
             {
-                <text>@title - </text>
+                if (Model.Content.Name.ToString().ToLower() == "documentation")
+                {
+                    var path = title.Split(new string[] { " - " }, StringSplitOptions.None);
+
+                    System.Globalization.TextInfo myTI = new System.Globalization.CultureInfo("en-US", false).TextInfo;
```

Comment:
Just use `var` and make sure to give your variable a meaningful name, I would suggest `textInfo`.